### PR TITLE
[TTAHUB-3513] Add goal template for monitor goal so we have an id to use in CRON

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

This simply add a goal template for the montior goal with the text from the jira. The idea here is once we have an ID it will never change and we can use it in the CRON job query.

## How to test

- Review the migration.
- Make sure the migration works in both directions.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3513

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: src/migrations/20241108102623-GrantRelationshipToActive.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const { prepMigration } = require('../lib/migration');
+
+const goalText = 'The recipient will develop and implement a CAP/QIP to address monitoring findings.';
```
Comment (GarrettEHill) [inline src/migrations/20241108102623-GrantRelationshipToActive.js:3]: Its missing the prefix, something like "(Monitoring)"

**Review by nvms — APPROVED** [review]

File: src/migrations/20241108102623-GrantRelationshipToActive.js (hunk 1 above)
Comment (AdamAdHocTeam) [reply src/migrations/20241108102623-GrantRelationshipToActive.js:3]: Sure it wasn't in the ticket, but I will add it.

File: src/migrations/20241108102623-GrantRelationshipToActive.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+const { prepMigration } = require('../lib/migration');
+
+const goalText = '(Monitoring) The recipient will develop and implement a CAP/QIP to address monitoring findings.';
```
Comment (GarrettEHill) [inline src/migrations/20241108102623-GrantRelationshipToActive.js:3]: That is not the posted standard goal
```suggestion
const goalText = '(Monitoring) The recipient will develop and implement a QIP/CAP to address monitoring findings.';
```

Comment (AdamAdHocTeam) [reply src/migrations/20241108102623-GrantRelationshipToActive.js:3]: This was copied directly from the JIRA ticket.

**Comment by AdamAdHocTeam** [comment]
> We understand that by merging this into main, this goal is now selectable by every user of the app? I thought this was supposed to be only for specific recipients

Per @GarrettEHill we need this in ahead of the cron job to ensure we have a template id to use in the job.

**Comment by thewatermethod** [comment]
That doesn't resolve the issue I brought up though, does it? It seems like if we need this goal in production now, we need to make additional provisions short of just adding it to the database. Unless I'm wrong about access to it.

**Pull request merged**